compositor_common/src/scene/common_transformations.rs

```diff
@@ -0,0 +1,22 @@
+use serde::{Deserialize, Serialize};
+
+use crate::util::RGBAColor;
+
+#[derive(Serialize, Deserialize, Clone)]
+#[serde(tag = "type", content = "params", rename_all = "snake_case")]
```

Comment:
```suggestion
#[serde(tag = "transformation", content = "params", rename_all = "snake_case")]
```
or operation, in general, let's try to avoid type if possible